### PR TITLE
Only Analyze projects passed in via CLI

### PR DESCRIPTION
**Pull request by TheAngryByrd**
Fix for #214

### CODE REVIEW

File: src/FSharp.Analyzers.Cli/Program.fs [hunk 1]
```diff
@@ -140,7 +142,12 @@ let loadProjects toolsPath properties (projPaths: string list) =
             logger.LogError("Failed to load project '{0}'", failedLoads)
             exit 1
 
-        return FCS.mapManyOptions projectOptions |> Seq.toList
+        let loaded =
+            FCS.mapManyOptions projectOptions
+            |> Seq.filter (fun p -> projPaths |> List.exists (fun x -> x = p.ProjectFileName)) // We only want to analyze what was passed in
```
Comment (nojaf) [inline src/FSharp.Analyzers.Cli/Program.fs:147]: Would this need a `Seq.distinctBy`?

**Comment by nojaf** [comment]
Could you add a release note / changelog entry? We can ship this fix as a revision.

**Comment by TheAngryByrd** [comment]
Done

**Review by nojaf — APPROVED** [review]
Thanks Jimmy!

**Pull request merged**